lib/index.js

```diff
@@ -0,0 +1,36 @@
+const getFoods = () => {
+return fetch(`http://localhost:3000/api/v1/foods`)
+  .then(response => handleResponse(response))
+  .then(foods => getAllFoods(foods))
+  .catch(error => console.log({error}));
+}
+const handleResponse = (response) => {
+  return response.json()
+  .then(json => {
+    if(!response.ok){
+      const error = {
+        status: response.status,
+        statusText: response.statusText,
+        json,
+      };
+      return Promise.reject(error);
+    }
+  return json;
+  })
+}
+const getAllFoods = (foods) => {
+
+  return foods.forEach((food) => {
```

Comment:
I think we can pull this out to two methods- one to iterate over the foods response, and another to actually append to the table and render on the page.

Reply:
Totally! I implemented this refactor - go ahead and merge if you like it!